DEVELOPING.md

```diff
@@ -52,22 +24,16 @@ make style
 make quality
 ```
 
-This will run automatic code styling using `black` and `isort` and test that the
+This will run automatic code styling using `ruff`, `flake8`, `black`, `isort`, and `mypy` to test that the
```

Comment:
should take out reference to mypy until we add it back in

Reply:
Good callout, fixed